docs/notebooks/doc2vec-IMDB.ipynb

```diff
@@ -610,7 +610,10 @@
     "        duration = 'na'\n",
     "        train_model.alpha, train_model.min_alpha = alpha, alpha\n",
     "        with elapsed_timer() as elapsed:\n",
-    "            train_model.train(doc_list, total_examples=train_model.corpus_count, epochs=train_model.iter)\n",
+    "            if not isinstance(train_model, ConcatenatedDoc2Vec):\n",
+    "              train_model.train(doc_list, total_examples=train_model.corpus_count, epochs=train_model.iter)\n",
+    "            else:\n",
+    "              train_model.train(doc_list)\n",
```

Comment:
The call doesn't have to use `train_model.corpus_count` from inside the model - it can just use `len(doc_list)`. And since the outside loop is handling the multiple passes, the `epochs` argument should be `1`.